bin/src/repo/license_repo.dart

```diff
@@ -0,0 +1,94 @@
+import 'dart:io';
+
+import 'package:http/http.dart';
+import 'package:path/path.dart';
+
+import '../model/dto/dependency.dart';
+import '../model/dto/dependency_license_data.dart';
+import '../model/dto/dependency_lock.dart';
+import '../model/pubspec.dart';
+import '../service/config_service.dart';
+import '../service/pubdev_webservice.dart';
+import '../util/logger.dart';
+
+const _flutterHomeUrl = 'https://flutter.dev/';
+const _flutterRepoUrl = 'https://github.com/flutter/flutter';
+const _flutterLicenseUrl = 'https://raw.githubusercontent.com/flutter/flutter/master/LICENSE';
+const _allowedLicenseFilesName = [
+  'LICENSE',
+  'LICENSE.md',
+  'LICENSE.txt',
+  'LICENSE.rst',
+  'License',
+  'license',
+  'License.md',
```

Comment:
Use case-insenstive compare instead of specifying casing

Reply:
This is not used for comparison It is used to check if that specific file exists. I could also write something that test every case of a specific string.

Reply:
/shrug

Reply:
For now it is fine :D but it could be improved in the future :)